cumulusci/tasks/salesforce/custom_settings_wait.py

```diff
@@ -106,7 +109,8 @@ def _apply_namespace(self):
 
     @property
     def success(self):
-        self.field_value = self.record[self.field_name]
+        lowerCaseRecord = {k.lower(): v for k, v in self.record.items()}
+        self.field_value = lowerCaseRecord[self.field_name.lower()]
```

Comment:
In Python we typically use snake case (underscores) for variables and reserve camel case for class names.

```suggestion
        lower_case_record = {k.lower(): v for k, v in self.record.items()}
        self.field_value = lower_case_record[self.field_name.lower()]
```